src/main/java/frc/robot/RobotContainer.java

```diff
@@ -37,17 +36,9 @@ public RobotContainer() {
   
  
   private void setUpDefaultCommands() {
-    drivetrain.setDefaultCommand(Commands.run(
-      () ->{
-        drivetrain.setLeftSpeed(MetersPerSecond.of(commandxboxcontroller.getLeftY() * MAX_SPEED_METERS_PER_SECOND));
-        drivetrain.setRightSpeed(MetersPerSecond.of(commandxboxcontroller.getRightY() * MAX_SPEED_METERS_PER_SECOND));
-        //drivetrain.setLeftSpeed(MetersPerSecond.of(MAX_SPEED_METERS_PER_SECOND));
-        //drivetrain.setRightSpeed(MetersPerSecond.of(MAX_SPEED_METERS_PER_SECOND));
-        System.out.println("yay we did it");
-  }
-      
-    , drivetrain)
-      );
+    drivetrain.setDefaultCommand(
+      drivetrain.tankDrive(() -> commandxboxcontroller.getRightY(), () -> commandxboxcontroller.getLeftY())
```

Comment:
negate the values from getRightY and getLeftY so that when those sticks are pushed forward, a positive value is sent to the tankDrive Command